common/src/main/java/org/conscrypt/OpenSSLCipherRSA.java

```diff
@@ -216,6 +216,26 @@ void engineInitInternal(int opmode, Key key, AlgorithmParameterSpec spec)
     }
 
     @Override
+    protected int engineGetKeySize(Key key) throws InvalidKeyException {
+        if (key instanceof OpenSSLRSAPrivateKey) {
+            return ((OpenSSLRSAPrivateKey) key).getModulus().bitLength();
+        } else if (key instanceof RSAPrivateCrtKey) {
```

Comment:
nit: simplify the code a bit by removing `else` throughout (since you're always returning)